src/components/Layout/LayoutView.js

```diff
@@ -24,6 +24,7 @@ import GlobalSearch from '../../pages/search/searchView';
 import GlobalSearchController from '../../pages/search/searchViewController';
 import Login from '../../pages/accessManagment/login';
 import userRegistration from '../../pages/accessManagment/userRegistration';
+import SysInfoView from '../../pages/sysInfo/view';
 
 // Access control imports
 import PrivateRoute, { LoginRoute } from './privateRoute';
```

Comment:
@ogwuazors There is ES error appearing in this branch after it's re-based. Can you fix it ?

Reply:
yea give me a moment